File: 03_Arreglos/Vectores/CSharp/ejemplo1.cs
using System;
class Programa {
    static void Main() {
        Console.Write("Ingrese N (tamaño del vector): ");
        int n = int.Parse(Console.ReadLine() ?? "0");
        int[] v = new int[n];

        for (int i = 0; i < n; i++) {
            Console.Write($"v[{i}] = ");
            v[i] = int.Parse(Console.ReadLine() ?? "0");
        }

        double suma = 0;
        foreach (int x in v) suma += x;
        double promedio = n > 0 ? suma / n : 0;

        int mayores = 0;
        foreach (int x in v) if (x > promedio) mayores++;

        Console.WriteLine($"\nPromedio = {promedio:F2}");
        Console.WriteLine($"Cantidad mayores al promedio = {mayores}");
    }
}

File: 03_Arreglos/Vectores/Python/ejemplo1.py
def main():
    n = int(input("Ingrese N (tamaño del vector): "))
    v = [int(input(f"v[{i}] = ")) for i in range(n)]

    promedio = sum(v)/n if n > 0 else 0
    mayores = sum(1 for x in v if x > promedio)

    print(f"\nPromedio = {promedio:.2f}")
    print(f"Cantidad mayores al promedio = {mayores}")

if __name__ == "__main__":
    main()
